capi/src/title_component_state.rs

```diff
@@ -46,8 +46,8 @@ pub extern "C" fn TitleComponentState_line1(this: &TitleComponentState) -> *cons
 pub extern "C" fn TitleComponentState_line2(this: &TitleComponentState) -> *const Nullablec_char {
     // FIXME: Add API for querying the abbreviations.
     this.line2
-        .as_ref()
-        .map_or_else(ptr::null, |abbrevs| output_str(&abbrevs.last().unwrap()))
+        .last()
+        .map_or_else(ptr::null, |abbrev| output_str(&abbrev))
```

Comment:
Is it ok that we can return null here but not in the TypeScript bindings?

Reply:
We don't use this function at all, since we serialize and deserialize the state to JSON. This is the alternative way if you don't serialize. This is what @DarkRTA is calling in his timer. The function here has no notion of abbreviations at all (because it was never introduced in this part of the C API), so it basically still only has an optional second line as a single string.